data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -368,15 +370,30 @@ fi
 
 echo "Starting cluster-bootstrap..."
 
-if [ ! -f cb-bootstrap.done ]
-then
-    bootkube_podman_run \
+run_cluster_bootstrap() {
+	bootkube_podman_run \
         --rm \
         --volume "$PWD:/assets:z" \
         --volume /etc/kubernetes:/etc/kubernetes:z \
         "${CLUSTER_BOOTSTRAP_IMAGE}" \
         start --tear-down-early=false --asset-dir=/assets --required-pods="${REQUIRED_PODS}"
+}
+
+gather_bootstrap_on_failure() {
+	/usr/local/bin/bootstrap-cluster-gather.sh
```

Comment:
Source a script with the function that you want to call and call that function rather than calling another script directly here.